src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.request;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderRequestDto {
+    private List<Long> ticketIds;
+    private LocalDateTime orderDate;
+    private Long userId;
+
+    public List<Long> getTicketIds() {
+        return ticketIds;
+    }
+
+    public void setTicketIds(List<Long> ticketIds) {
+        this.ticketIds = ticketIds;
+    }
+
+    public LocalDateTime getOrderDate() {
+        return orderDate;
+    }
+
+    public void setOrderDate(LocalDateTime orderDate) {
+        this.orderDate = orderDate;
+    }
+
+    public Long getUserId() {
+        return userId;
+    }
+
+    public void setUserId(Long userId) {
+        this.userId = userId;
+    }
+}
```

Comment:
`Don’t create redundant dtos and mappers for them.` - from CM, this will be reject reason.
Or show me where do you use it.